--- /opt/svnadmin/classes/util/global.func.php
+++ global.func.php
@@ -1,25 +1,7 @@
 <?php
 function checkPHPVersion($minimumVersion)
 {
-  $phpVersion = phpversion();
-  $phpVersionParts = explode(".", $phpVersion);
-  $minVersionParts = explode(".", $minimumVersion);
-  $minVersionPartsCount = count($minVersionParts);
-
-  $check = true;
-  if ($minVersionPartsCount >= 1)
-    if ($phpVersionParts[0] < $minVersionParts[0])
-      $check = false;
-
-  if ($minVersionPartsCount >= 2)
-    if ($phpVersionParts[1] < $minVersionParts[1])
-      $check = false;
-
-  if ($minVersionPartsCount >= 3)
-    if ($phpVersionParts[2] < $minVersionParts[2])
-      $check = false;
-
-  return $check;
+    return version_compare(PHP_VERSION, $minimumVersion);
 }

 function exception_handler($exception)
 